master/docs/manual/cfg-statustargets.rst

```diff
@@ -1478,10 +1478,13 @@ GerritStatusPush
 :class:`GerritStatusPush` sends review of the :class:`Change` back to the Gerrit server, optionally also sending a message when a build is started.
 GerritStatusPush can send a separate review for each build that completes, or a single review summarizing the results for all of the builds.
 
-.. py:class:: GerritStatusPush(server, username, reviewCB, startCB, port, reviewArg, startArg, summaryCB, summaryArg, ...)
+.. py:class:: GerritStatusPush(server, username, notify, reviewCB, startCB, port, reviewArg, startArg, summaryCB, summaryArg, ...)
 
    :param string server: Gerrit SSH server's address to use for push event notifications.
    :param string username: Gerrit SSH server's username.
+   :param string notify: Notify handling that defines to whom email notifications should be sent after the review is stored.
```

Comment:
It is available in the official documentation: https://gerrit-documentation.storage.googleapis.com/Documentation/2.11/cmd-review.html